spec/devengo/api/accounts_service_spec.rb

```diff
@@ -44,7 +45,8 @@
                     available_cents: 10_000,
                     total_cents: 11_000,
                     bank: Devengo::Resources::Shared::ThirdPartyBank,
-                    metadata: { example_key: "example_value" }
+                    metadata: { example_key: "example_value" },
+                    created_at: "2023-01-01T12:00:00Z"
```

Comment:
It is not necessary to define the value of `created_at` in each test case, they have the same value for all of them, therefore, we can add the value in the `shared_example`.